cmd/go-filecoin/mining.go

```diff
@@ -149,30 +149,17 @@ var stringEncoderMap = cmds.EncoderMap{
 	}),
 }
 
-// MiningAddPieceResult is a wrapper around the uint64 sectorID
-type MiningAddPieceResult struct {
```

Comment:
I probably could have left Add-Piece as is, but it's not doing much good like this. It might be worth implementing it next.